src/d_iwad.c

```diff
@@ -939,6 +939,22 @@ const char *D_SaveGameIWADName(GameMission_t gamemission)
     // This ensures that doom1.wad and doom.wad saves are stored
     // in the same place.
 
+    if (gamevariant == freedoom)
+    {
+        if (gamemission == doom)
+        {
+            return "freedoom1.wad";
+        }
+        else if (gamemission == doom2)
+        {
+            return "freedoom2.wad";
+        }
+    }
+    else if (gamevariant == freedm && gamemission == doom2)
+    {
+        return "freedoom2.wad";
```

Comment:
Why not `freedm.wad` here?

Reply:
I can do that. I guess I was confused if FreeDM was its own class of IWAD entirely or like an extension of freedoom2

Reply:
I'd say it's a separate IWAD on its own.